site/docs/starter/slash-commands.md

```diff
@@ -0,0 +1,550 @@
+---
+sidebar_position: 5
+---
+
+# Step 5 - Registering and Handling Slash Commands
+
+As was mentioned in step 3, discord recommends that you implement slash command support in your bot. This is done via
+the interaction system. We'll also need to have a list of the guilds our bot is in, so we'll add the discordeno cache
+plugin to do this for us.
+
+## Cache Plugin
+
+The [Cache Plugin](https://github.com/discordeno/discordeno/blob/main/plugins/cache) provides a simple way to cache
+users and guilds in memory. We need to add it into our dependencies:
+
+```typescript title="deps.ts" {3}
+export * from "https://deno.land/x/discordeno@v13.0.0-rc45/mod.ts";
+export { config as dotEnvConfig } from "https://deno.land/x/dotenv@v3.1.0/mod.ts";
+export * from "https://deno.land/x/discordeno@13.0.0-rc45/plugins/mod.ts";
+```
+
+Then enable the plugin as well as it's sweepers:
+
+```typescript title="mod.ts" {5,8,23,30}
+// Imports
+import { logger } from "./src/utils/logger.ts";
+import {
+  dotEnvConfig,
+  enableCachePlugin, 
+  createBot, 
+  GatewayIntents, 
+  enableCacheSweepers, 
+  startBot 
+} from "./deps.ts";
+import { events } from "./src/events/mod.ts";
+
+const log = logger({ name: "Main" });
+
+// Secrets
+dotEnvConfig({ export: true });
+export const BOT_TOKEN = Deno.env.get("BOT_TOKEN") || "";
+export const BOT_ID = BigInt(atob(BOT_TOKEN.split(".")[0]));
+
+log.info("Starting Bot, this might take a while...");
+
+// Create the Bot Object
+export const bot = enableCachePlugin(createBot({
+  token: BOT_TOKEN,
+  botId: BOT_ID,
+  intents: GatewayIntents.Guilds | GatewayIntents.GuildMembers | GatewayIntents.GuildMessages | GatewayIntents.DirectMessages,
+  events
+}));
+
+enableCacheSweepers(bot);
+
+if (BOT_TOKEN) {
+  // Start the bot
+  await startBot(bot);
+} else {
+  throw "Error: No Token!";
+}
+```
+
+This then exposes `bot.guilds`, `bot.members`, and more. The sweepers will clean the cache to make sure we aren't using
```

Comment:
I believe this is a big feature in our lib, unlike others. Sweepers' goal is to prevent a memory leak that will crash ur process if u keep caching everything and never clean it out ur process will eventually run out of memory.

Reply:
Should I add this reason?